src/App.tsx

```diff
@@ -1,9 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+export type Movie = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+};
+
 export const App: React.FC = () => {
+  const [query, setQuery] = useState<string>('');
+
+  const filterMovies = (list: Movie[]) => {
+    let tmp = [...list];
+
+    if (query.length > 0) {
+      tmp = tmp.filter(
+        movie =>
+          movie.title
+            .toLocaleLowerCase()
+            .includes(query.toLocaleLowerCase().trim()) ||
+          movie.description
+            .toLocaleLowerCase()
+            .includes(query.toLocaleLowerCase().trim()),
```

Comment:
[checklist](https://github.com/mate-academy/react_movies-list-filter/blob/master/checklist.md): Don't repeat yourself. If you see that you copy/paste some block code, maybe, it's time to create a separate variable/function